zerver/models.py

```diff
@@ -852,10 +852,10 @@ class UserProfile(AbstractBaseUser, PermissionsMixin):
     EMBEDDED_BOT = 4
 
     BOT_TYPES = {
-        DEFAULT_BOT: 'Generic bot',
-        INCOMING_WEBHOOK_BOT: 'Incoming webhook',
-        OUTGOING_WEBHOOK_BOT: 'Outgoing webhook',
-        EMBEDDED_BOT: 'Embedded bot',
+        DEFAULT_BOT: str(_('Generic bot')),
+        INCOMING_WEBHOOK_BOT: str(_('Incoming webhook')),
+        OUTGOING_WEBHOOK_BOT: str(_('Outgoing webhook')),
+        EMBEDDED_BOT: str(_('Embedded bot')),
```

Comment:
Why the `str()` here

Reply:
I wasn't sure how to tag these strings for translation. `_('Embedded bot')` did not work, and caused my dev env to crash. I checked the translation documentation [here](https://zulip.readthedocs.io/en/latest/translating/internationalization.html#backend-translations), but there wasn't an example that could clarify this. I have reverted this commit for now, and pushed a fix for the View source/Edit button.